internal/pkg/cli/flag.go

```diff
@@ -188,18 +189,23 @@ Supported providers are: %s.`, strings.Join(manifest.PipelineProviders, ", "))
 )
 
 const (
-	appFlagDescription      = "Name of the application."
-	envFlagDescription      = "Name of the environment."
-	svcFlagDescription      = "Name of the service."
-	jobFlagDescription      = "Name of the job."
-	workloadFlagDescription = "Name of the service or job."
-	nameFlagDescription     = "Name of the service, job, or task group."
-	pipelineFlagDescription = "Name of the pipeline."
-	profileFlagDescription  = "Name of the profile."
-	yesFlagDescription      = "Skips confirmation prompt."
-	execYesFlagDescription  = "Optional. Whether to update the Session Manager Plugin."
-	jsonFlagDescription     = "Optional. Outputs in JSON format."
-	forceFlagDescription    = "Optional. Force a new service deployment using the existing image."
+	appFlagDescription        = "Name of the application."
+	envFlagDescription        = "Name of the environment."
+	svcFlagDescription        = "Name of the service."
+	jobFlagDescription        = "Name of the job."
+	workloadFlagDescription   = "Name of the service or job."
+	nameFlagDescription       = "Name of the service, job, or task group."
+	pipelineFlagDescription   = "Name of the pipeline."
+	profileFlagDescription    = "Name of the profile."
+	yesFlagDescription        = "Skips confirmation prompt."
+	execYesFlagDescription    = "Optional. Whether to update the Session Manager Plugin."
+	jsonFlagDescription       = "Optional. Outputs in JSON format."
+	forceFlagDescription      = "Optional. Force a new service deployment using the existing image."
+	noRollbackFlagDescription = `Optional. 
+Disable automatic stack rollback in case 
+of a failure during the deployment. 
+We do NOT recommend using this flag if 
+you are deploying to a prod environment.`
```

Comment:
nit:
```suggestion
	noRollbackFlagDescription = `Optional. 
Disable automatic stack rollback in case 
of a deployment failure. We do not recommend 
using this flag for a production environment.`
```